.github/actions/build-fixtures/action.yaml

```diff
@@ -24,6 +24,15 @@ runs:
       id: evm-builder
       with:
         type: ${{ steps.properties.outputs.evm-type }}
+    - name: Wrap ethereum/tests fixtures with eofwrap tool
+      shell: bash
+      if: ${{ steps.properties.outputs.eofwrap }}
+      run: |
+        curl -L https://github.com/ethereum/tests/archive/refs/tags/v14.1.tar.gz | tar -xz
```

Comment:
I'll un-hardcode the `14.1` too maybe.

Reply:
That's a good idea. How about adding it to `.github/configs/feature.yaml`?